crates/rune-cli/Cargo.toml

```diff
@@ -21,7 +21,7 @@ codespan-reporting = "0.9.5"
 anyhow = "1.0.32"
 bincode = "1.3.1"
 log = "0.4.11"
-
+argh = "0.1.3"
```

Comment:
I keep the internal deps one line separate, because it makes it easier to verify version bumps during releases. Please keep it for now.